bitnami/elasticsearch/templates/data-statefulset.yaml

```diff
@@ -216,18 +216,22 @@ spec:
 {{- if not .Values.data.persistence.enabled }}
         - name: "data"
           emptyDir: {}
+{{- else if .Values.data.persistence.existingClaim }}
+        - name: "data"
+          persistentVolumeClaim:
+            claimName: {{ .Values.data.persistence.existingClaim }}
 {{- else }}
   volumeClaimTemplates:
     - metadata:
         name: "data"
         {{- if .Values.data.persistence.annotations }}
         annotations: {{- toYaml .Values.data.persistence.annotations | nindent 10 }}
-        {{- end }}
+        {{- end }}        
       spec:
         accessModes: {{- toYaml .Values.data.persistence.accessModes | nindent 10 }}
         {{ $storage := dict "global" .Values.global "local" .Values.data }}
         {{ include "elasticsearch.storageClass" $storage }}
         resources:
           requests:
             storage: {{ .Values.data.persistence.size | quote }}
-{{- end }}
+{{- end }}
```

Comment:
Could you fix the "No newline at end of file" error?

Reply:
Passed the linter, so I'm not sure where this error is. I'll add an extraneous new line to both files I changed. Maybe that will solve it?

Reply:
fixed